static/src/components/LoginView.vue

```diff
@@ -0,0 +1,59 @@
+<script lang="ts">
+import { defineComponent } from "vue";
+import { LoginService as API } from "@/api/services/LoginService";
+
+export default defineComponent({
+  emits: ["login"],
+
+  data() {
+    return {
+      /**
+       * Content of the username filed
+       */
+      username: "",
+      /**
+       * Content of the password field
+       */
+      password: "",
+    };
+  },
+
+  methods: {
+    async loginUser() {
+      if (this.username != "" && this.password != "") {
+        try {
+          await API.login({
+            username: this.username,
+            password: this.password,
+          });
+          this.$emit("login");
+        } catch {
+          this.$toast?.danger("Username or Password is not correct...");
+        }
+      } else {
+        this.$toast?.warning("Username and Password needs to be set...");
+      }
+    },
+  },
+});
+</script>
+
+<template>
+  <div>
+    <input
+      v-model="username"
+      type="text"
+      name="username"
+      placeholder="Username"
+    />
+    <br />
+    <input
+      v-model="password"
+      type="password"
+      name="password"
+      placeholder="Password"
+    />
+    <br />
+    <button id="submit" type="button" @click="loginUser">Login!</button>
```

Comment:
I know, you dont like Inkline :D
It probably breaks the tests, right?
If it doesnt, would you like to consider using [i-input](https://www.inkline.io/docs/forms/form) and [i-button](https://www.inkline.io/docs/components/button)
And apart from that, I think it would be nice if the login form is centered in the screen. What do you think?

Reply:
Should definitely be done! It's rather like this: I don't like front-end styling xD
I could switch it to i-input and i-button respectively and see whether the tests still work afterwards. But I have no clue how to center it, I would need your help for that :sweat_smile:

Reply:
It's i-input / i-button now, only not yet centered